src/controllers/expenses.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const {
+  getAllExpenses,
+  addExpense,
+  getExpense,
+  deleteExpense,
+  updateExpense,
+} = require('../services/exprensesService');
+
+function getAllExpensesAction(req, res) {
+  const expenses = getAllExpenses();
+
+  res.send(expenses);
+};
+
+function addExpenseAction(req, res) {
+  const expense = req.body;
+  const listPlaces = ['title', 'amount', 'category', 'note'];
+  const keysExpense = Object.keys(expense);
+  const { userId } = req.body;
+
+  const missingFields = listPlaces
+    .filter(field => !keysExpense.includes(field));
+
+  if (missingFields.length
```

Comment:
The status code 422 is not correct for missing fields or invalid userId. Please use 400 status code for bad request.

Reply:
done